dScripts/EquipmentTriggers/CoilBackpackBase.cpp

```diff
@@ -14,10 +14,10 @@ void CoilBackpackBase::NotifyHitOrHealResult(Entity* self, Entity* attacker, int
 		if (self->GetVar<uint8_t>(u"coilCount") > 4) {
 			auto* skillComponent = self->GetComponent<SkillComponent>();
 			if (!skillComponent) return;
-			skillComponent->CalculateBehavior(m_SkillId, m_BehaviorId, self->GetObjectID());
+			skillComponent->CastSkill(m_SkillId);
```

Comment:
Functional change, the target will now be empty instead of self

Reply:
Maybe this was intentional

Reply:
It was wrong and cause the telsa pack to stack AOE by how many enemies were found by the aoe behavior. This let's it function as intended

Reply:
Over 2000 damage was caused by 4 triggers of the Tesla backpack. Which it should only do 3 per trigger. Tested the other backpacks as well to make sure they still worked